data/json/items/containers.json

```diff
@@ -689,10 +665,10 @@
     "name": { "str": "large cardboard box", "str_pl": "large cardboard boxes" },
     "looks_like": "box_medium",
     "description": "A very large cardboard box, the sort children would have loved to hide in, when there were still children.",
-    "weight": "1250 g",
-    "volume": "325000 ml",
-    "longest_side": "90 cm",
-    "//": "A box that children could hide in should be big enough to cram a (un)dead child or uncomfortable adult in (90x90x60)",
+    "weight": "5 kg",
```

Comment:
These weights seem really high to me. Where did you get these values, boxes I'm seeing on Uhaul aren't close to this weight.

Reply:
I got the weights from taking the medium cardboard box, assuming the resources used to craft was more or less correct, then scaling up or down from there, based on the relative size and area. 

I should have checked for cardboard boxes more carefully, as they are quite standardized. 

https://www.uhaul.com/MovingSupplies/Boxes/Double-Wall-Boxes/Heavy-Duty-Extra-Large-Moving-Box-Double-Walled-24-x-18-x-24 
This is a slightly smaller size to the large cardboard box, and weighs slightly under 2 kg, so I will scale by a factor of 2.5.